### ISSUE TITLE
Change `.addCommand` to `sop.command` to match Vorpal API.

### ISSUE THREAD

**Issue by dthree** [AljoschaMeyer/vorpal-setorprint#1]
IMHO I think you should make it `sop.command`, as then it will match Vorpal's API. Might make it more intuitive.

Also, perhaps the options should be the second param, similar to how Vorpal declares the command first and then options.

**Comment by AljoschaMeyer** [comment]
Maintaining an analogy to vorpal's api is a good idea, I'll do that.

I actually planned changing the method signature to sop.command(obj, key, options), with options taking the command-specific description, validation method, printing method and - planned feature - a callback when successfully setting a value, each defaulting to one of the global options.

I do need the first two arguments to be object and key though. Or at least I didn't find a better way of implementing this.

**Comment by dthree** [comment]
Nice. 

Maybe just `sop.command(key, obj[, options])`. I just think key should be first IMHO.

**Comment by AljoschaMeyer** [comment]
Might as well add a callback when failing validation. That way the validation method does not need to e.g. notify the user.

**Comment by AljoschaMeyer** [comment]
Ok, commited the changes. Mind taking a look at the new readme? I'm still not sure how to clearly communicate what the module does exactly. But the new api seems nice.

To further behave like `vorpal.command()`, I'll look into having `sop.command()` return the command, to allow chaining just like vorpal, e.g.

``` coffee
sop.command key, obj
  .alias 'foobar'
  .description 'I like trains.'
```

Edit: Turns out this was already the current behavior, coffeescript is just that good :)

**Comment by dthree** [comment]
Nice!!

A few of your examples still have it the other way around: `object, key`. After  you straighten those out, I think we're good to go!

**Comment by AljoschaMeyer** [comment]
Done and released as 1.0.0.

As you can chain vorpals methods on the command, I also removed the description option. No need to devote extra code to that now.

**Comment by dthree** [comment]
Sweet!
